src/lib/desktop.ts

```diff
@@ -625,3 +626,19 @@ export const getUser = async (
   }
   return Promise.reject(new Error('unreachable'))
 }
+
+export const writeProjectThumbnailFile = async (
+  dataUrl: string,
+  projectDirectoryPath: string
+) => {
+  const filePath = window.electron.path.join(
+    projectDirectoryPath,
+    PROJECT_IMAGE_NAME
+  )
+  const data = atob(dataUrl.substring('data:image/png;base64,'.length))
+  const asArray = new Uint8Array(data.length)
+  for (let i = 0, len = data.length; i < len; ++i) {
+    asArray[i] = data.charCodeAt(i)
+  }
+  return window.electron.writeFile(filePath, asArray)
+}
```

Comment:
Would probably be good to add a unit test for this guy. But since we didn't want to spend too too much time on this it's probably fine as-is.